.github/workflows/production_build_and_deploy.yaml

```diff
@@ -0,0 +1,60 @@
+name: Production build and Deploy
+
+on:
+  push:
+    branches: master
```

Comment:
Could you do on release maybe? It shouldn't change anything to the rest of the yaml

Reply:
Yeah sure, do you mean on tag?

Reply:
There should be on push release I believe. That's what I had done on combiner

Reply:
Ok cool I'll give that a go